simpleSequential/test_stringToInteger.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from stringToInteger import stringToInteger
+
+class TestStringToInteger(unittest.TestCase):
+    def test_positiveStringInput(self):
+        self.assertEqual(stringToInteger("5"), 5)
+
+    def test_negativeStringInput(self):
+        self.assertEqual(stringToInteger("-6"), -6)
+
+    def test_zeroNumberInput(self):
```

Comment:
Let this be “test_zeroNumberNumericalStringInput”

Reply:
Did you read this one? zero number input makes me think of "no input", which is not what you mean.

Reply:
I think you fix the wrong file. Did you check the file name?